README.md

```diff
@@ -0,0 +1,135 @@
+# Java Function Invoker [![Build Status](https://travis-ci.org/projectriff/java-function-invoker.svg?branch=master)](https://travis-ci.org/projectriff/java-function-invoker)
+
+## Install as a riff invoker
+
+```bash
+riff invokers apply -f java-invoker.yaml
+```
+
+## Development
+
+Testing locally:
+
+```bash
+./mvnw clean install dockerfile:build
+docker run -ti -p 8080:8080 -v `pwd`/target/test-classes:/classes projectriff/java-function-invoker:0.0.6-snapshot --function.uri='file:classes?handler=io.projectriff.functions.Doubler'
+```
+
+Then
+
+```bash
+curl -v localhost:8080 -H "Content-Type: text/plain" -d 5
+10
+```
+
+
+## riff Commands
+
+- [riff init java](#riff-init-java)
+- [riff create java](#riff-create-java)
+
+<!-- riff-init -->
+
+### riff init java
+
+Initialize a java function
+
+#### Synopsis
+
+Generate the function based on the function source code specified as the filename, using the artifact (jar file),
+the function handler(classname), the name and version specified for the function image repository and tag.
+
+For example, from a maven project directory named 'greeter', type:
+
+    riff init java java -i greetings -a target/greeter-1.0.0.jar --handler=Greeter
```

Comment:
java java?

Reply:
that's a bug in the current java-invoker.yaml, will fix